pdns/recursordist/docs/lua-config/protobuf.rst

```diff
@@ -12,8 +12,27 @@ Configuring Protocol Buffer logs
 --------------------------------
 Protobuf export to a server is enabled using the ``protobufServer()`` directive:
 
+.. function:: protobufServer(server [, options]))
+
+  .. versionadded:: 4.2.0
+
+  :param string server: The IP and port to connect to
+  :param table options: A table with key: value pairs with options.
+
+  Options:
+
+  * ``timeout=2``: int - Time in seconds to wait when sending a message
+  * ``maxQueuedEntries=100``: int - How many entries will be kept in memory if the server becomes unreachable
+  * ``reconnectWaitTime=1``: int - How long to wait, in seconds, between two reconnection attempts
+  * ``taggedOnly=false``: bool - Only entries with a policy or a policy tag set will be sent
+  * ``asyncConnect``: bool - When set to false (default) the first connection to the server during startup will block up to ``timeout`` seconds, otherwise the connection is done in a separate thread, after the first message has been queued
+  * ``logQueries=true``: bool - Whether to export queries
+  * ``logResponses=true``: bool - Whether to export responses
+
```

Comment:
Maybe adding a reference to `setProtobufMasks()` here would be great.

Reply:
Just a curiosity, is there an option (would it be useful) to anonymize IPs using `ipcipher` ?

Reply:
> Maybe adding a reference to `setProtobufMasks()` here would be great.

Done!

> Just a curiosity, is there an option (would it be useful) to anonymize IPs using `ipcipher`?

There isn't. While I totally agree it would be nice to have it, it's probably out of scope for this PR.